### ISSUE TITLE
track status of doctest on all Octave doc strings

### ISSUE THREAD

**Issue by mtmiller** [catch22/octave-doctest#188]
Is there or should there be an issue or wiki tracking the status of all of Octave's core doc strings passing with doctest? I'm thinking checklists of doctest issues, upstream issues, individual Octave doc strings that need work. Could be tracked here, on a wiki, or more awkwardly on savannah.

I just edited about 20 doc strings to pass with doctest for Octave 5 (https://hg.savannah.gnu.org/hgweb/octave/rev/f6730533820e), and I think it would be useful to have this as a goal documented somewhere.

**Comment by cbm755** [comment]
It might be good to try a release candidate for 0.7.0 on Octave just to make sure we haven't broken anything that would hinder this effort...

@mtmiller how do you actually do this?  In the source tree after building?
```
make install
./run-octave
>> pkg install doctest-x.y.z+
>> doctest .
```

I get a lot of stuff like
```
/tmp/octave-help-OOYWOO:702: unknown command `DOCSTRING'
/tmp/octave-help-OOYWOO:704: unknown command `DOCSTRING'
/tmp/octave-help-OOYWOO:706: unknown command `DOCSTRING'
/tmp/octave-help-OOYWOO:708: unknown command `DOCSTRING'
/tmp/octave-help-OOYWOO:710: unknown command `DOCSTRING'
```
And maybe we shouldn't be testing both `.txi` and `.texi`?  Just the `.texi` files which are autogenerated from `.txi`?

**Comment by mtmiller** [comment]
Yeah, I don't really remember how I did it before, but here's a way that works whether or not Octave is already installed, I think this gets pretty close to all documented functions. In the build tree

    doctest ([__builtins__; 'libinterp/dldfcn'; '../scripts'])

or in a running installed Octave

    doctest ([__builtins__; __octave_config_info__('octfiledir'); __octave_config_info__('fcnfiledir')])

**Comment by cbm755** [comment]
Thanks!  I tried the 0.7.0 RC with Octave 6.0.0 install,, checked out today:

 * `__builtins__`: hangs on popen2
 * `__octave_config_info__('octfiledir')`:  FAIL 1/1
 * ` __octave_config_info__('fcnfiledir')`:  FAIL 74/214

With doctest-0.6.1:

 * `__builtins__`: hangs on popen2
 * `__octave_config_info__('octfiledir')`:  N/A (not supported)
 * ` __octave_config_info__('fcnfiledir')`:  FAIL 73/214

So no significant differences between 0.6.1 and 0.7.0, which is good.
